index.html

```diff
@@ -1,66 +1,354 @@
+                  <li class="tag">
+                    <a class="tagText" id="html">html</a>
+                  </li>
+                  <li class="tag">
+                    <a class="tagText" id="bootstrap">bootstrap</a>
+                  </li>
+                  <li class="tag">
+                    <a class="tagText" id="Ruby">Ruby</a>
+                  </li>
+                </ul>
+            </ul>
+            <div id="cardButton">
+              <button class="button" type="button">
+                <span id="buttonText">See Project</span>
+              </button>
+            </div>
+          </div>
+          <div class="flex-container" id="cardWork">
+            <ul id="cardElements">
+              <li>
+                <h1 id="cardTitle">Profesional Art Printing Data</h1>
+              </li>
+              <li>
+                <h1 id="cardDesc">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</h1>
+              </li>
+              <li id="cardTags">
+                <ul class="flex-container" id="tagGroup">
+                  <li class="tag">
+                    <a class="tagText" id="html">html</a>
+                  </li>
+                  <li class="tag">
+                    <a class="tagText" id="bootstrap">bootstrap</a>
+                  </li>
+                  <li class="tag">
+                    <a class="tagText" id="Ruby">Ruby</a>
+                  </li>
+                </ul>
+            </ul>
+            <div id="cardButton">
+              <button class="button" type="button">
+                <span id="buttonText">See Project</span>
+              </button>
+            </div>
+          </div>
+          <div class="flex-container" id="cardWork">
+            <ul id="cardElements">
+              <li>
+                <h1 id="cardTitle">Profesional Art Printing Data</h1>
+              </li>
+              <li>
+                <h1 id="cardDesc">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</h1>
+              </li>
+              <li id="cardTags">
+                <ul class="flex-container" id="tagGroup">
+                  <li class="tag">
+                    <a class="tagText" id="html">html</a>
+                  </li>
+                  <li class="tag">
+                    <a class="tagText" id="bootstrap">bootstrap</a>
+                  </li>
+                  <li class="tag">
+                    <a class="tagText" id="Ruby">Ruby</a>
+                  </li>
+                </ul>
+            </ul>
+            <div id="cardButton">
+              <button class="button" type="button">
+                <span id="buttonText">See Project</span>
+              </button>
+            </div>
+          </div>
+      </div>
+      <div class="flex-container" id="aboutMyself">
+        <ul class="flex-container" id="aboutMe">
           <li>
```

Comment:
```html
 <div class="flex-container" id="aboutMyself">
```

- Please let's make the container of the section a `section` tag with the reason explained above.

Your design
![image](https://user-images.githubusercontent.com/78251927/212380260-0d78b821-9803-4b06-bae7-157f73329e54.png)


Figma
![image](https://user-images.githubusercontent.com/78251927/212380166-4b9ccd34-c192-4ced-807a-1bd778e40980.png)

- Please kindly include the missing items 
  - The section background image
  - The horizontal line in the design